pkgs/by-name/mi/minimeters/package.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, appimageTools
+, requireFile
+, unzip
+, autoPatchelfHook
+, openssl
+, SDL2
+, pulseaudio
+, freetype
+}:
+
+stdenv.mkDerivation {
+  pname = "minimeters";
+  version = "0.8.8";
+
+  src = requireFile {
+    name = "MiniMeters-Linux-v0.8.8.zip";
+    hash = "sha256-6Gi3Y1WWOyU5o/qwPTEV0MkbDnGm/n9P4GWQKM8v/iU=";
+    url = ''https://directmusic.itch.io/minimeters (Legacy versions, extract "MiniMeters 0.8.8.zip")'';
```

Comment:
This is not a pure URL. Would it be clearer to explain this in `message`?

Reply:
I agree that'd be better. I'll do this, but it might be a while until I can.